whisper/src/bin/whisper-merge.rs

```diff
@@ -0,0 +1,52 @@
+#[macro_use]
+extern crate structopt;
+extern crate failure;
+extern crate whisper;
+
+use failure::Error;
+use std::path::PathBuf;
+use std::process::exit;
+use std::time::{SystemTime, UNIX_EPOCH};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "whisper-merge")]
+struct Args {
+    /// Path to data file
+    #[structopt(name = "from_path", parse(from_os_str))]
+    from_path: PathBuf,
+
+    /// Path to data file
+    #[structopt(name = "to_path", parse(from_os_str))]
+    to_path: PathBuf,
+
+    /// Begining of interval, unix timestamp(default: epoch)
+    #[structopt(long = "from")]
+    from: Option<u32>,
+
+    /// End of interval, unix timestamp (default: now)
+    #[structopt(long = "until")]
+    until: Option<u32>,
+}
+
+fn main() -> Result<(), Error> {
+    let args = Args::from_args();
+
+    for filename in &[&args.from_path, &args.to_path] {
```

Comment:
i think here better to se `is_file` because it check for existance and wether it really file
https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.is_file